const Attempt = () => {
  return (
    <>
      <div>
        <div>
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. A atque
            perspiciatis quisquam ipsa nemo molestiae quis corporis accusamus
            illum deleniti odit natus architecto veritatis fugit in blanditiis,
            ex modi dolore. Harum facilis ab deserunt perspiciatis doloremque
            vitae autem magni, minima omnis officiis provident modi eius iusto
            iste non possimus soluta, cum dignissimos in tempora voluptatum!
            Asperiores incidunt suscipit illum culpa?
          </p>
        </div>
        <div>
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod ipsum,
            laboriosam rem unde facere eaque excepturi sequi fugit tenetur modi,
            quos dolorum! Molestias illum laborum facere dolorem alias. Ad,
            dolorum. Non quaerat consequatur fugiat dicta tempore iure
            consequuntur numquam, quas, tenetur voluptatum ratione cum explicabo
            enim itaque eius debitis qui eligendi veritatis, consectetur
            officiis quia. Voluptas perspiciatis mollitia doloremque aut.
            Praesentium autem rerum provident iure maxime dolorem soluta,
            accusamus aspernatur dolore consectetur possimus ipsum assumenda
            alias architecto qui explicabo voluptates voluptatibus magni iusto
            earum omnis cumque reprehenderit. Consequuntur, perspiciatis nulla.
            Laudantium, esse obcaecati magni magnam vero reprehenderit corporis,
            qui quia explicabo dolor nemo repudiandae, fuga minus libero
            molestias? Iste inventore corrupti sint nobis quo ab nesciunt minus
            enim recusandae quod?
          </p>
        </div>
      </div>
    </>
  );
};

export default Attempt;
